code.go

```diff
@@ -1400,3 +1400,9 @@ func Of(name string) (code Code, ok bool) {
 	}
 	return Identity, false
 }
+
+// Register adds a user defined code to the multicodec table.
+func Register(code Code, name string) error {
```

Comment:
This file is auto-generated. This code needs to be put in `gen.go`.

Alternatively, even better, just create a separate "utils.go" file that's not auto-generated and move the `Of` func there (and stop auto-generating that.